fashion-messaging-app-api/server.js

```diff
@@ -91,52 +91,58 @@ app.get('/posts', async (req, res) => {
     res.status(500).json({ message: err.message });
   }
 });
-app.get('/photos', async (req, res) => {
-  const query = req.query.search || defaultQuery;
 
-  
-  try {
-    const response = await clientAPI.photos.search({ query, per_page: PHOTOS, page: 1});
+    app.get('/photos', async (req, res) => {
+      const query = req.query.search || defaultQuery;
+      const perPage = (query === defaultQuery) ? 20 : PHOTOS; 
```

Comment:
This conditional helps me to not return too many photos in a fetch request. If it is the default query, it will return 20 objects

Reply:
How you choose 20 as a limit ?